mirrord/intproxy/src/proxies/incoming/http.rs

```diff
@@ -53,26 +53,29 @@ pub async fn handshake(
 impl HttpSender {
     pub async fn send(
         &mut self,
-        req: HttpRequestFallback,
+        request: HttpRequestFallback,
     ) -> InterceptorResult<Response<Incoming>, InterceptorError> {
+        let request = if let HttpRequestFallback::Chunked(_req, replay) = request {
+            Request::builder()
+                .version(Version::HTTP_11)
+                .body(StreamBody::new(replay.stream()).boxed())
+                .unwrap()
```

Comment:
Gotta add more stuff like uri and headers
I think now that `HttpRequestFallback::Chunked` variant holds the replay stream, we can move this logic into `HttpRequestFallback::into_hyper`? It's ok to consume the value (given that replay stream implements `Clone` in correct way)